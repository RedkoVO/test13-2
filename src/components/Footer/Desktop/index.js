import React from 'react'
import PropTypes from 'prop-types'
import withStyles from '@material-ui/core/styles/withStyles'
import { Link } from 'react-router-dom'
import cn from 'classnames'

import styles from './styles'

const Footer = ({ classes }) => (
  <footer className={classes.root}>
    <ul className={classes.footerNav}>
      <li>
        <Link to="/" className={classes.footerNavItem}>
          Dashboard
        </Link>
      </li>
      <li>
        <Link to="/" className={classes.footerNavItem}>
          Wallet
        </Link>
      </li>
      <li>
        <Link to="/" className={classes.footerNavItem}>
          Shop
        </Link>
      </li>
      <li>
        <Link to="/games" className={classes.footerNavItem}>
          Games
        </Link>
      </li>
      <li>
        <Link to="/" className={classes.footerNavItem}>
          Friends
        </Link>
      </li>
      <li>
        <Link
          to="/"
          className={cn(classes.footerNavItem, classes.footerNavSettings)}
        >
          Settings
        </Link>
      </li>
    </ul>
    <span className={classes.copyrightGameFooter}>
      Vault. All Rights Reserved 2018
    </span>
  </footer>
)

Footer.propTypes = {
  classes: PropTypes.object
}

export default withStyles(styles)(Footer)
