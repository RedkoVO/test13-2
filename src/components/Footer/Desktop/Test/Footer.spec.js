import * as React from 'react'
import renderer from 'react-test-renderer'
import { MemoryRouter } from 'react-router-dom'

import Footer from '../'

it('renders correctly', () => {
  const tree = renderer
    .create(
      <MemoryRouter keyLength={0}>
        <Footer />
      </MemoryRouter>
    )
    .toJSON()
  expect(tree).toMatchSnapshot()
})
