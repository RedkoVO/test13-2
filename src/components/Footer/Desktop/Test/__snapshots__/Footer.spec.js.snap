// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<footer
  className="Footer-root-1"
>
  <ul
    className="Footer-footerNav-2"
  >
    <li>
      <a
        className="Footer-footerNavItem-3"
        href="/"
        onClick={[Function]}
      >
        Dashboard
      </a>
    </li>
    <li>
      <a
        className="Footer-footerNavItem-3"
        href="/"
        onClick={[Function]}
      >
        Wallet
      </a>
    </li>
    <li>
      <a
        className="Footer-footerNavItem-3"
        href="/"
        onClick={[Function]}
      >
        Shop
      </a>
    </li>
    <li>
      <a
        className="Footer-footerNavItem-3"
        href="/games"
        onClick={[Function]}
      >
        Games
      </a>
    </li>
    <li>
      <a
        className="Footer-footerNavItem-3"
        href="/"
        onClick={[Function]}
      >
        Friends
      </a>
    </li>
    <li>
      <a
        className="Footer-footerNavItem-3"
        href="/"
        onClick={[Function]}
      >
        Settings
      </a>
    </li>
  </ul>
  <span>
    Vault. All Rights Reserved 2018
  </span>
</footer>
`;
